From e7401f0eb805223fa866d27975f7054d8059f608 Mon Sep 17 00:00:00 2001
From: manexwayo <wayomanasseh@gmail.com>
Date: Thu, 9 Jul 2020 02:33:19 +0100
Subject: [PATCH] enable sudoless build

---
 build/tools/build_helper | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/build/tools/build_helper b/build/tools/build_helper
index d5fbf40..8af0a9a 100644
--- a/build/tools/build_helper
+++ b/build/tools/build_helper
@@ -38,7 +38,7 @@ case "$OS_DISTRO" in
   ubuntu) OS_BASEDISTRO="debian"; INSTALLER="apt-get"; CMAKE="cmake" ;;
 esac
 
-SUDO='sudo -S -E'
+SUDO=''
 
 ###############################
 ## echo and  family
-- 
2.25.1.windows.1

