From 89c70dfb493c203977837a9191c0832d96d9710e Mon Sep 17 00:00:00 2001
From: manexwayo <wayomanasseh@gmail.com>
Date: Sun, 28 Jun 2020 20:05:11 +0100
Subject: [PATCH 3/3] remove unfound library

---
 cmake_targets/CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/cmake_targets/CMakeLists.txt b/cmake_targets/CMakeLists.txt
index e10311a..8b54a4c 100644
--- a/cmake_targets/CMakeLists.txt
+++ b/cmake_targets/CMakeLists.txt
@@ -2123,7 +2123,7 @@ else()
   message("No Blas/Atlas libs found, some targets will fail")
 endif()
 
-list(APPEND ATLAS_LIBRARIES lapack lapacke)
+list(APPEND ATLAS_LIBRARIES lapack)
 
 set(XFORMS_SOURCE
   ${OPENAIR1_DIR}/PHY/TOOLS/lte_phy_scope.c
-- 
2.25.1.windows.1

